### PR TITLE
fix(nu-command/tests): further remove unnecessary pipeline() and cwd()

### PR DESCRIPTION
**Pull request by harshalchaudhari35**
# Description

This PR further fixes tests as part of #8670 

# User-Facing Changes

None

# Tests + Formatting

None

# After Submitting

None

### CODE REVIEW

File: crates/nu-command/tests/commands/every.rs [hunk 1]
```diff
@@ -194,15 +134,7 @@ fn skips_every_third_row() {
...
-        let actual = nu!(
-            cwd: dirs.test(), pipeline(
-            r#"
-                ls
-                | get name
-                | every 3 --skip
-                | to json --raw
-            "#
-        ));
+        let actual = nu!( cwd: dirs.test(), " ls | get name | every 3 --skip | to json --raw ");
```
Comment (sholderbach) [inline crates/nu-command/tests/commands/every.rs:137]: I have to say I find it harder to read when everything is crammed on to one line, especially when I try to see the difference between two test cases at a glance.

If it makes sense removing the multiline-raw-string the white-space around should possibly go as well?

Comment (rgwood) [reply crates/nu-command/tests/commands/every.rs:137]: I agree. The intent of #8670 was to clean up tests where `pipeline()` is not doing anything useful; I'm OK leaving `pipeline()` calls in where they make the code more readable.

Comment (sholderbach) [reply crates/nu-command/tests/commands/every.rs:137]: I think the problem is not the removal of pipeline directly but rather the change away from the multiline strings. (might be that rustfmt starts to do its magic if it is removed)

Comment (harshalchaudhari35) [reply crates/nu-command/tests/commands/every.rs:137]: Yeah fair point. I was on the fence on cramming it in one line also. Will revert any obvious multi-line cases. Although, I'm not sure why the white-spaces around the quotes are still persisting even after I ran `cargo fmt --all`

Comment (sholderbach) [reply crates/nu-command/tests/commands/every.rs]: Thanks!

Comment (harshalchaudhari35) [reply crates/nu-command/tests/commands/every.rs]: Should be good to now I think, CI is passing too

**Review by rgwood — APPROVED** [review]
LGTM, thanks!

**Pull request merged**